ext/storage_sources/gcp_store/gcp_connection.cpp

```diff
@@ -93,10 +92,9 @@ gcp_connection::delete_object(const std::string &object_key)
 {
     auto status = _gcp_client.DeleteObject(_bucket_name, _object_prefix + object_key);
 
-    if (!status.ok()) {
-        std::cerr << status.message() << std::endl;
-        return -1;
-    }
+    if (!status.ok())
+        return handle_error(status, "Delete '" + object_key + "' failed: " + status.message());
```

Comment:
Do we know what `status.message()` looks like? In other words, is adding the `"Delete '" + object_key + "' failed:` redundant?

Reply:
Hi Tammy,
Examples of status.message() are as follows:
`"Permanent error DeleteObject: No such object: quickstart_test/prefixadasddeletethis.txt"`

`"UploadFileResumable(ResumableUploadRequest={bucket_name=unit_testing_gcp, object_name=gcptest-unit-2023-02-01-03-43-09/361067088--test_non_exist, disable-md5-hash=1, auto-finalize=enabled}, test_non_exist.txt): cannot open upload file source"`

I added the `"Delete '" + object_key + "' failed:` because I recalled finding error messages that weren't clear when working on other tickets.

 I cannot seem to reproduce them now however and I'm happy to just print out the status.message() if you agree with this change.

Reply:
I think it's fine as is if it helps the user. We can revisit after testing if the error messages seem redundant.